resources/datomic/schema.dtm

```diff
@@ -522,34 +552,100 @@
   :db/ident       :stats/wood-barriers-destroyed
   :db/valueType   :db.type/long
   :db/cardinality :db.cardinality/one
-  :db/doc         "Number of wood-barriers destroyed"}
+  :db/doc         "Number of wood barriers destroyed"}
```

Comment:
So technically you can shoot a barrier multiple times before it's destroyed (another player could be shooting it from the other side) so those values would be different.